docs/requirements-doc.txt

```diff
@@ -15,7 +16,7 @@ sphinx-autodoc-typehints>=1.21.7
 sphinx-design>=0.3.0
 pydantic
 # TODO: separate document for pyodide-build?
-pyodide-build>=0.27.3
+-e ./pyodide-build
```

Comment:
I think this would break the Dockerfile as we don't copy the submodules there (I use that locally, at least)

Reply:
We can fix the Dockerfile later.